src/ArduinoIoTCloudLPWAN.cpp

```diff
@@ -149,8 +149,9 @@ void ArduinoIoTCloudLPWAN::sendPropertiesToCloud()
 {
   int bytes_encoded = 0;
   uint8_t data[CBOR_LORA_MSG_MAX_SIZE];
+  static unsigned int last_checked_property_index = 0;
```

Comment:
This now is where you save the state for the `CBOREncoder::encode` state machine. I'd suggest to rather store it as a protected (not private, because both `ArduinoIoTCloudLPWAN` and `ArduinoIoTCloudTCP` need to access it) **member variable** of class `ArduinoIoTCloud`. Otherwise you've just created a hidden state variable. In my experience, everything that controls state in a C++ class should be listed as a member in the class body.

Reply:
now i got it! plese see https://github.com/arduino-libraries/ArduinoIoTCloud/pull/293/commits/5576a608dba2186e87c8cdc8f8f81be42a0687fd